src/components/cards/AirdropCard.svelte

```diff
@@ -104,7 +104,8 @@
 						</div>
 					</aside>
 				{/if}
-				{#if thisDrop.claim.dates.claimDateClose >= new Date() - 7 * 24 * 60 * 60 * 1000}
+				{#if thisDrop.claim.dates.claimDateClose === null}
```

Comment:
i'm not sure what the end date is, but without tweaking the template here it said the claim was closed

Reply:
Nice!